subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AvailableJavaHomes.java

```diff
@@ -243,21 +241,23 @@ private DevInfrastructureJvmLocator(FileCanonicalizer fileCanonicalizer) {
         public List<JvmInstallation> findJvms() {
             List<JvmInstallation> jvms = new ArrayList<>();
             if (OperatingSystem.current().isLinux()) {
-                jvms = addJvm(jvms, JavaVersion.VERSION_1_5, "1.5.0", new File("/opt/jdk/sun-jdk-5"), true, i386);
-                jvms = addJvm(jvms, JavaVersion.VERSION_1_6, "1.6.0", new File("/opt/jdk/sun-jdk-6"), true, x86_64);
-                jvms = addJvm(jvms, JavaVersion.VERSION_1_6, "1.6.0", new File("/opt/jdk/ibm-jdk-6"), true, x86_64);
-                jvms = addJvm(jvms, JavaVersion.VERSION_1_7, "1.7.0", new File("/opt/jdk/oracle-jdk-7"), true, x86_64);
-                jvms = addJvm(jvms, JavaVersion.VERSION_1_8, "1.8.0", new File("/opt/jdk/oracle-jdk-8"), true, x86_64);
-                jvms = addJvm(jvms, JavaVersion.VERSION_1_9, "1.9.0", new File("/opt/jdk/oracle-jdk-9"), true, x86_64);
+                addJvm(jvms, JavaVersion.VERSION_1_6, "1.6.0", new File("/opt/jdk/sun-jdk-6"));
+                addJvm(jvms, JavaVersion.VERSION_1_6, "1.6.0", new File("/opt/jdk/ibm-jdk-6"));
+                addJvm(jvms, JavaVersion.VERSION_1_7, "1.7.0", new File("/opt/jdk/oracle-jdk-7"));
+                addJvm(jvms, JavaVersion.VERSION_1_8, "1.8.0", new File("/opt/jdk/oracle-jdk-8"));
+                addJvm(jvms, JavaVersion.VERSION_1_9, "1.9.0", new File("/opt/jdk/oracle-jdk-9"));
+            } else if(OperatingSystem.current().isWindows()) {
+                addJvm(jvms, JavaVersion.VERSION_11, "1.8", new File("C:\\Program Files\\Java\\jdk1.8"));
+                addJvm(jvms, JavaVersion.VERSION_11, "11", new File("C:\\Program Files\\Java\\jdk-11"));
```

Comment:
This is because https://builds.gradle.org/viewLog.html?buildId=37128644&buildTypeId=Gradle_Check_Quick_2_bucket20&tab=buildResultsDiv